Makefile

```diff
@@ -0,0 +1,53 @@
+PELICANOPTS=
+
+BASEDIR=$(CURDIR)
+INPUTDIR=$(BASEDIR)/content
+OUTPUTDIR=$(BASEDIR)/output
```

Comment:
What we usually do for this is to use a script, as described in the pelican docs: http://docs.getpelican.com/en/3.6.3/tips.html#publishing-to-github